python/eos/analysis.py

```diff
@@ -191,22 +203,24 @@ def optimize(self, start_point=None, **kwargs):
         else:
             eos.info('Optimization goal achieved after {nfev} function evaluations'.format(nfev=res.nfev))
 
-        for p, v in zip(self.varied_parameters, res.x):
+        bfp = self.__x_to_par(res.x)
+
+        for p, v in zip(self.varied_parameters, bfp):
             p.set(v)
 
-        return eos.BestFitPoint(self, res.x)
+        return eos.BestFitPoint(self, bfp)
 
 
     def log_pdf(self, x, *args):
         """
         Adapter for use with external optimization software (e.g. pypmc) to aid when optimizing the log(posterior).
 
-        :param x: Parameter point, with the elements in the same order as in eos.Analysis.varied_parameters.
+        :param x: Parameter point, with the elements in the same order as in eos.Analysis.varied_parameters, rescaled between -1 and 1.
```

Comment:
``` Python
:param x: Parameter point, with the elements in the same order as in eos.Analysis.varied_parameters,
rescaled so that every element is in the interval [-1, +1].
```